include/grpc/impl/codegen/grpc_types.h

```diff
@@ -235,6 +235,12 @@ typedef struct {
 /** Should we allow receipt of true-binary data on http2 connections?
     Defaults to on (1) */
 #define GRPC_ARG_HTTP2_ENABLE_TRUE_BINARY "grpc.http2.true_binary"
+/** An experimental channel arg which configures the perferred frame size http2
+ * setting sent to the peer at startup. If set to -1 (default), the preferred
+ * frame size is not sent to the peer. Otherwise it is clamps the specified
+ * value between 16384 and 16777215. */
+#define GRPC_ARG_EXPERIMENTAL_HTTP2_PREFERRED_FRAME_SIZE \
```

Comment:
let's rename this `GRPC_ARG_EXPERIMENTAL_HTTP2_PREFERRED_CRYPTO_FRAME_SIZE` and refer everywhere to this as crypto frame size - since it's a different frame size to the http2 frame size and this could get confusing.

Reply:
Done.

Reply:
Wait. I didn't edit it as crypto frame size everywhere. I'll do that in next commit